packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -28,6 +28,10 @@ export const getComboboxStyles = (theme: GrafanaTheme2) => {
       borderRadius: theme.shape.radius.default,
       overflow: 'hidden',
     }),
+    scrollContainer: css({
```

Comment:
think this can just go, right? 🤔

Reply:
yes, I found that ScrollContainer extended BoxProps :)